### DISCUSSION TITLE
Log cleanup on the pull request screen

### DISCUSSION THREAD

**Discussion by 911432** [community/community#53751] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

1. I wish I could clear the past history when I changed the title.
2. When force-pushing an open pull request, prevent previous force-push history from being displayed. This is because if you did the push now, you don't need to show the previous push. Or I wish the writers could erase their respective histories.

**Comment by oorzkws** [comment]
This would also be nice across pull requests. I reference another open PR in my commit message, but it I force push, it spams the comments of said linked PR 😞
